test/src/main/java/org/jvnet/hudson/test/PluginAutomaticTestBuilder.java

```diff
@@ -73,9 +78,27 @@ private static boolean notSkipTests(String propertyName) {
         return !Boolean.getBoolean("hpiTest.skip"+propertyName);
     }
 
-    public static class CliSanityTest extends HudsonTestCase {
```

Comment:
I would keep a test suite, because there can be more CLI-oriented tests later. A hierarchical approach is better IMO even if it impacts the tests execution speed